packages/ui-components/src/components/Code/Code.component.tsx

```diff
@@ -14,16 +13,16 @@ const codeStyles = `
 /** A basic inline <code> component.
  *   Accepts "content" prop or renders children as passed.
  */
-export const Code = ({ content = "", children = null, className = "", ...props }) => {
+export const Code: React.FC<CodeProps> = ({ content = "", children = null, className = "", ...props }) => {
   return (
     <code className={`juno-code ${codeStyles} ${className}`} {...props}>
-      {content || children}
+      {(content || children) as React.ReactNode}
```

Comment:
I don't think we need to cast here? What do you think? 👀

Reply:
Right